src/styles/blocks/_card.scss

```
@@ -0,0 +1,123 @@
+@use "./variables" as *;
+
+.container {
+    display: grid;
+    grid-template-columns: repeat(1, $card-width);
+    margin: auto;
+    gap: 46px 48px;
+    padding: 50px 40px;
+    justify-content: center;
+    align-items: center;
+}
+
+.card {
+    width: 200px;
+    padding: 32px 16px 16px;
+    display: flex;
+    flex-direction: column;
+    border: 1px solid $grey-color;
+    border-radius: 5px;
+}
+
+.card__image {
+    width: 160px;
+    height: 134px;
+    background-repeat: no-repeat;
+    background-size: contain;
+    align-self: center;
+}
+
+.card__nameProduct {
+    font-size: 12px;
+    font-weight: 500;
+    margin-top: 40px;
+    line-height: 18px;
+    color: $main-color;
+    letter-spacing: 0;
```

card id your block so you can use nesting of scss
```suggestion
.card {
  &__image {...}
  &__nameProduct {..}
    ...
}
```